Content.Server/DeltaV/Speech/EntitySystems/GermanAccentSystem.cs

```diff
@@ -0,0 +1,33 @@
+using Content.Server.DeltaV.Speech.Components;
+using Content.Server.Speech;
+using Content.Server.Speech.EntitySystems;
+using System.Text.RegularExpressions;
+
+namespace Content.Server.DeltaV.Speech.EntitySystems;
+
+public sealed class GermanAccentSystem : EntitySystem
+{
+    [Dependency] private readonly ReplacementAccentSystem _replacement = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<GermanAccentComponent, AccentGetEvent>(OnAccentGet);
+    }
+
+    // converts left word when typed into the right word. For example typing you becomes ye.
+    public string Accentuate(string message, GermanAccentComponent component)
+    {
+        var msg = message;
+
+        msg = _replacement.ApplyReplacements(msg, "german");
```

Comment:
This line specifically can be replaced with a couple uses of String.Replace, which would make the Replacements unnecessary entirely.

Reply:
This how accent replacements should be done